### DISCUSSION TITLE
Editor plugins

### DISCUSSION THREAD

**Discussion by nephix** [avitorio/outstatic#20] [Ideas]
Hey

Are there any extension points or plugins that I an hook into the editor?

Looking to add video embedding and code with syntax highlights

Any pointers into the right direction welcome 🙏

**Comment by avitorio** [comment]
Hey @nephix 
Right now the library doesn't have a streamlined way of adding extensions or plugins to the editor. But it's something we definitely want to do.

We use [TipTap](https://tiptap.dev/) as our editor. If you want you can try adding those features.

Let me know how it goes!

**Reply by avitorio** [reply]
By the way @nephix,

Syntax highlight should already be available inside the code editor.

**Comment by DCzajkowski** [comment]
Adding support for [GitHub Flavored Markdown](https://github.github.com/gfm/) would be great! Currently, some of our posts use tables, but those are impossible to work on in the editor (and the editor breaks them, by putting rows in one line).

**Reply by avitorio** [reply]
I've looked into adding support for tables but didn't go too far into it. It is definitely a good feature to add.
